static/templates/tab_bar.handlebars

```diff
@@ -1,21 +1,17 @@
 <ul id="tab_list">
-    {{#each tabs}}
-    <li class="{{active}} {{cls}}" {{#if data}}data-name="{{data}}"{{/if}}>
-        {{#if icon}}
-        <i class="fa fa-angle-right" aria-hidden="true"></i>
-        {{/if}}
```

Comment:
From what I understand from the git history, we stopped sending "icon" to this template 6 years ago in commit `a6486a1fa`, and yet this was never removed from the template itself. This is partially why I moved towards such a refactor.